### ISSUE TITLE
Auto suggest 

### ISSUE THREAD

**Issue by Thijzer** [Peekmo/atom-symfony2#7]
Question, could you execute console commands in atom? Auto suggest a database update via notifications after adjusting an entity might be the way to go for a lazy programmer.

**Comment by Peekmo** [comment]
Auto suggest is not a bad idea, I lik it. But console commands in this plugin is not a so usefull feature. There's some plugins to have a terminal in atom, and it's far better in my opinion, 'cause in many symfony2 commands, you have to interact with the program.

**Comment by Thijzer** [comment]
True, a separate drum computer emulator panel connected to the symfony  commands  is what we need. :) I just hate typing those commands all the time.
